iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/Analog.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface Analog {
+
+    enum Side {
+        ONE,
+        TWO,
+        THREE
+    }
+
+    enum Type {
+        THREE_PHASE_POWER,
```

Comment:
I don't get these types. Can you start with `ACTIVE_POWER`, `REACTIVE_POWER`, `CURRENT`. For `VOLTAGE` and `PHASE`, it means that it will be attached to a connectable. It is what we have from CGMES conversion too? Are we obliged to extend a connectable? Just a suggestion: could it be possible to have just an extension of network or voltage level?

Reply:
I honestly have no idea, but those are the standard types in CGMES. We can not keep them but in the CGMES conversion, we will have to think about how to translate those types in powsybl's type.